compose/ts-translation-service.yml

```diff
@@ -0,0 +1,38 @@
+version: '2.4'
+
+services:
+  ts-translation-service:
+    build:
+      context: .
+      args:
+        - http_proxy
+        - https_proxy
+        - no_proxy
+    image: "${TS_TRANSLATION_SERVICE_LOCAL-hmctspublic.azurecr.io/}/ts/translation-service:${TS_TRANSLATION_SERVICE_TAG:-latest}"
```

Comment:
Can we rename `TS_TRANSLATION_SERVICE_LOCAL` to `TS_TRANSLATION_SERVICE_USE_LOCAL` to match the other services in backend.yml.

Reply:
done